packages/react-params/src/Param/findComponent.ts

```diff
@@ -60,7 +62,7 @@ const componentDef: TypeToComponent[] = [
   { c: Text, t: ['String', 'Text'] },
   { c: Struct, t: ['Struct'] },
   { c: Tuple, t: ['Tuple'] },
-  { c: Vector, t: ['Vec'] },
+  { c: Vector, t: ['Vec', 'VecDeque'] },
```

Comment:
Would not add this, the API does not handle these at all (so there is no proper TypeDef for these for actual proper parsing) - just for the simple reason that it is not in Substrate master.

Reply:
ok